app/src/main/java/org/simple/clinic/questionnaireresponse/sync/QuestionnaireResponsePullResponse.kt

```diff
@@ -0,0 +1,16 @@
+package org.simple.clinic.questionnaireresponse.sync
+
+import com.squareup.moshi.Json
+import com.squareup.moshi.JsonClass
+import org.simple.clinic.sync.DataPullResponse
+
+@JsonClass(generateAdapter = true)
+data class QuestionnaireResponsePullResponse(
+
+    @Json(name = "questionnaire_responses")
+    override val payloads: List<QuestionnaireResponsePayload>,
+
+    @Json(name = "process_token")
+    override val processToken: String
```

Comment:
In the `adding-sync-resource.md` guide, there's a Step 3. which says:
> Save the `processToken` in a shared preference and provide it through the Dagger DI graph.

Are we doing this? We couldn't find it.

Reply:
@olttwa yes we are doing this.The code can be found at `QuestionnaireResponseModule.kt`